packages/roosterjs-editor-plugins/lib/test/fragmentHandlerTest.ts

```diff
@@ -118,6 +118,26 @@ describe('fragmentHandler', () => {
                 );
             });
+            // e.g.
+            //text text
+            // .a
+            //   .b
+            //     1.c1
+            //     2.c2
+            //   .d
+            //text text
+            // -------------- --------------
+            //| text text    | text text    |
+            // -------------- --------------
+            //| .a           | .a           |
+            // -------------- --------------
+            it('fragments contains text, list and table that consist of list', () => {
+                runTest(
+                    '<html><body><div class="OutlineElement"><div class="TableContainer"><table><tbody><tr><td><div><div class="OutlineElement"><p><span><span>asdfasdf</span></span><span></span></p></div></div></td><td><div><div class="OutlineElement"><p><span><span>asdfasdf222</span></span><span></span></p></div></div></td></tr><tr><td><div><div class="ListContainerWrapper"><ul><li role="listitem" data-aria-level="1" class="OutlineElement">A</li></ul></div></div></td><td><div><div class="ListContainerWrapper"><ul><li role="listitem" data-aria-level="1" class="OutlineElement">A</li></ul></div></div></td></tr></tbody></table></div></div></body></html>',

```

Comment:
I believe you can simplify it more. Such as 
```html
<p><span><span>asdfasdf</span></span><span></span></p>
```
should be simplified to
```html
<p>asdfasdf</p>
```
Same for others